docs/rtd_environment.yml

```diff
@@ -13,10 +13,12 @@ dependencies:
   - sphinxcontrib-bibtex
   - sphinx-copybutton
   - sphinx-panels
+  - sphinx-design
   - ipython
   - ipython_genutils
   - myst-nb
   - pydata-sphinx-theme
```

Comment:
Two things:

1. Can you do the same changes in the central [`environment.yml`](https://github.com/OpenSourceEconomics/estimagic/blob/main/environment.yml)
2. Can't we remove `pydata_sphinx_theme` from the environment?